Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/api/ui/TestForAPIFragment.kt

```diff
@@ -222,6 +220,7 @@ class TestForAPIFragment : Fragment(R.layout.fragment_test_for_a_p_i),
         binding.apply {
             buttonApiTestStart.setOnClickListener { start() }
             buttonApiGetExposureKeys.setOnClickListener { getExposureKeys() }
+            buttonApiShareRandomIds.setOnClickListener { getExposureKeys() }
```

Comment:
Hi @ralfgehrer  

so there a button _buttonApiGetExposureKeys_ that already handles the method _getExposureKeys_

Just wanted to provide you this info.

Reply:
Thanks for pointing that out. Will clarify ...

Reply:
Changes are up. Please review when you can